src/index.js.flow

```diff
@@ -106,6 +106,27 @@ export type Beverage = {
   year: ?number,
 };
 
+export type BeverageMutator = {
+  abv: ?number,
+  availableId: ?EntityID,
```

Comment:
@jlkalberer  fix this on the server to `availabilityId` please

Reply:
File an issue